src/Kafka/CreateKafkaInstance/components/CloudProviderTiles.tsx

```diff
@@ -1,37 +1,68 @@
-import { Skeleton, Tile } from "@patternfly/react-core";
+import {
+  FormSelect,
+  FormSelectOption,
+  SelectProps,
+  Skeleton,
+  Tile,
+} from "@patternfly/react-core";
 import { AzureIcon } from "@patternfly/react-icons";
 import AwsIcon from "@patternfly/react-icons/dist/esm/icons/aws-icon";
-import { Fragment, VoidFunctionComponent } from "react";
+import { VoidFunctionComponent } from "react";
 import { Provider, ProviderInfo } from "../machines";
+import "./CloudProviderTiles.css";
+import { useTranslation } from "react-i18next";
 
 export type CloudProvidersTileProps = {
   providers: ProviderInfo[];
   value: Provider | undefined;
   onChange: (provider: Provider) => void;
   isDisabled?: boolean;
+  validated?: SelectProps["validated"];
 };
 export const CloudProvidersTiles: VoidFunctionComponent<
   CloudProvidersTileProps
-> = ({ providers, value, onChange, isDisabled }) => {
-  if (providers.length === 0) {
-    return (
-      <Skeleton className="pf-m-text-4xl" screenreaderText="Loading contents" />
-    );
-  }
-
+> = ({ providers, value, onChange, isDisabled, validated }) => {
+  const { t } = useTranslation();
   return (
     <>
+      {providers.length === 0 && (
+        <Skeleton
+          className="pf-m-text-4xl"
+          screenreaderText="Loading contents"
+        />
+      )}
       {providers.map(({ id, displayName }) => (
-        <Fragment key={id}>
-          <Tile
-            title={displayName}
-            icon={tiles[id]}
-            isSelected={value === id}
-            isDisabled={isDisabled}
-            onClick={() => onChange(id)}
-          />{" "}
-        </Fragment>
+        <Tile
+          key={id}
+          className={"mas--CreateKafkaInstance__CloudProviderTile"}
+          title={displayName}
+          icon={tiles[id]}
+          isSelected={value === id}
+          isDisabled={isDisabled}
+          onClick={() => onChange(id)}
+        />
       ))}
+      <FormSelect
```

Comment:
Since we are using the `Tile` component as a form field, we need to make it behave like a form field. The trick I used here is to have a shadow (hidden) select component that mimicks what the show in the tiles, to make this accessible.